src/Strategy/CommonClassesStrategy.php

```diff
@@ -82,13 +83,25 @@
                 'condition' => [\Buzz\Client\FileGetContents::class, \Buzz\Message\ResponseBuilder::class],
             ],
         ],
+        Psr18Client::class => [],
     ];
 
     /**
      * {@inheritdoc}
      */
     public static function getCandidates($type)
     {
+        if (Psr18Client::class === $type) {
+            $candidates = self::$classes[$type];
```

Comment:
i would prefer to say `$candiates = self::$classes[PSR18Client::class];` to be more explicit as to what we do here.

Reply:
Good!